app/api/routes.py

```diff
@@ -914,6 +920,32 @@
     '/user-favourite-events/<int:id>/relationships/event',
 )
 
+# user favourite sessions
+api.route(
+    UserFavouriteSessionListPost, 'user_favourite_session_post', '/user-favourite-session'
```

Comment:
No need of fetching all favourites

Reply:
? It's a post request to create a favourite session

Reply:
OK, got it